roles/hammer_devel/defaults/main.yml

```diff
@@ -14,3 +14,9 @@ hammer_devel_repositories:
   - theforeman/hammer-cli-foreman-tasks
   - katello/hammer-cli-csv
   - katello/hammer-cli-katello
+hammer_devel_local_gems: |-
+  gem 'hammer_cli'
+    gem 'hammer_cli_katello'
```

Comment:
Intending looks a bit off

Reply:
The indentation is correct to format the file correctly. Try it! :)

Reply:
I'll point you to this very confusing documentation http://yaml.org/spec/1.2/spec.html#id2777534